lms/djangoapps/grades/signals/handlers.py

```diff
@@ -56,13 +58,14 @@ def submissions_score_set_handler(sender, **kwargs):  # pylint: disable=unused-a
     if user is None:
         return
 
-    PROBLEM_WEIGHTED_SCORE_CHANGED.send(
+    PROBLEM_WEIGHTED_SCORE_CHANGED.send_robust(
```

Comment:
This change is the other that helped with my lettuce test issues - `send_robust` will send the signal to all handlers of a given signal, regardless of whether there was an exception in an earlier handler. Does anyone object to this change here? It seems smart, given that otherwise we could run into an issue where a poorly-written signal handler gets run first and crashes at a point we aren't expecting.